tools/amd_build/pyHIPIFY/cuda_to_hip_mappings.py

```diff
@@ -1732,6 +1727,11 @@
     ("cublasCgemm3mStridedBatched", ("rocblas_cgemm_3m_strided_batched", CONV_MATH_FUNC, API_BLAS, HIP_UNSUPPORTED)),
     ("cublasZgemmStridedBatched", ("rocblas_zgemm_strided_batched", CONV_MATH_FUNC, API_BLAS, HIP_UNSUPPORTED)),
     ("cublasHgemmStridedBatched", ("rocblas_hgemm_batched", CONV_MATH_FUNC, API_BLAS, HIP_UNSUPPORTED)),
+    ("cublasSgemm", ("rocblas_sgemm", CONV_MATH_FUNC, API_BLAS)),
+    ("cublasDgemm", ("rocblas_dgemm", CONV_MATH_FUNC, API_BLAS)),
+    ("cublasCgemm", ("rocblas_cgemm", CONV_MATH_FUNC, API_BLAS)),
```

Comment:
No, it is not. when hipifying caffe2 files, whole word is not searched for. For examples `cublasSgemmBatched` gets mapped to `rocblas_sgemmBatched` instead of `rocblas_sgemm_batched` if just `cublasSgemm` mapping takes place as it before in the list. Hence order matters. On the upside, this reduces the number of mappings.

Reply:
I think the root fix for this could be in the logic that does the string substitution, assuming all the substitutions here are for "whole words"

Reply:
Substitutions are not for whole words.